### ISSUE TITLE
Sentry plugin does not work. The annotation is not picked up

### ISSUE THREAD

**Issue by bob-bins** [backstage/backstage#6429]
I followed [these steps](https://www.npmjs.com/package/@backstage/plugin-sentry) exactly, even adding the "demo mode". Then I added the `sentry.io/project-slug` annotation to my component config. I also made another unrelated change to the config to make sure I knew the changes are being picked up. But somehow I am still seeing `The sentry.io/project-slug annotation is missing. You need to add the annotation to your component if you want to enabled for this tool.` wherever the Sentry data should be.

## Expected Behavior

I expected to see the Sentry demo data

## Current Behavior

I see `The sentry.io/project-slug annotation is missing. You need to add the annotation to your component if you want to enabled for this tool.`

## Possible Solution

No idea

## Steps to Reproduce

Follow [these steps](https://www.npmjs.com/package/@backstage/plugin-sentry) from the sentry-plugin NPM package, enabling "demo mode"

**Comment by bob-bins** [comment]
I've downgraded to `0.3.13` and it's working now

**Comment by benjdlambert** [comment]
Hmm - that's very strange as it's only dependencies that got bumped.

Did you use `yarn backstage-cli versions:bump` to bump the sentry package or did you just do it manually?

**Comment by bob-bins** [comment]
I did it manually. Deleted the lockfile and changed the package.json version to be exactly `0.3.13`.

**Comment by benjdlambert** [comment]
Yeah I think the issue is that you would need to bump other dependencies too. Editing the lockfile could cause duplicate packages to be brought in, which could cause unexpected things to happen. If you use the `yarn backstage-cli version:bump` command it will do this de-dupe for you.

**Comment by benjdlambert** [comment]
Try with that command and see if it helps

**Comment by bob-bins** [comment]
You might have missed this comment I left below the main message: https://github.com/backstage/backstage/issues/6429#issuecomment-878502128

Everything worked after I downgraded one patch version from the latest by editing the package.json file and regenerating a new lockfile (I didn't hand-edit the lockfile).

**Comment by warmfusion** [comment]
I'm seeing something similar with 0.3.18

is there some way to debug this further?

Interestingly, i'm also getting a similar error with the Sonar plugin too - perhaps theres something more fundemental going on?

**Comment by warmfusion** [comment]
I stand corrected - I'm not sure why, how or otherwise, but the plugin is now operational. I wish i could tell you what changed, but alas no such luck.
Sorry for the false negative.

**Comment by freben** [comment]
@bob-bins Are you still having this issue even with recent versions of Backstage?

**Comment by bob-bins** [comment]
It has been working for me after I downgraded `plugin-sentry` from 0.3.14 to 0.3.13 and redeployed. Not sure what the real root cause was however

**Comment by erikvanbrakel** [comment]
I had a similar issue with another plugin (sonarqube), and running `yarn install` to make sure all dependencies are set up properly did the trick.

**Comment by Rugvip** [comment]
I have a feeling this is due to multiple versions of `@backstage/plugin-catalog-react`. We'll likely bring in cross-version compatibility like we've done for `@backstage/core-plugin-api`, but for now the fix is to deduplicate the installations of that package, with yarn resolutions if needed.
